src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactHeaderSection/ContactHeaderEmailSection.tsx

```diff
@@ -35,9 +35,15 @@ export const ContactHeaderEmailSection = ({
         <TextSkeleton variant="text" />
       </ContactHeaderSection>
     );
-  } else if (contact) {
+  } else if (
+    contact &&
+    contact.primaryPerson &&
+    contact.primaryPerson.primaryEmailAddress
```

Comment:
Use optional chaining (`?.`) here. Or just replace the destructuring below with it.

This also applies to the `primaryPhoneNumber` version.